### PR TITLE
FRT-27 users catalog

### PR DESCRIPTION
**Pull request by BigiDigi89**
[FRT-27](https://aya-soft.atlassian.net/browse/FRT-27)
1. Customized admin dictionary with font-awesome
2. Added functionality to confirm user's email by admin
3. Added functionality to assign user as admin

### CODE REVIEW

File: app/controllers/admin/set_confirm_email_controller.rb [hunk 1]
```diff
@@ -0,0 +1,19 @@
+class Admin::SetConfirmEmailController < ApplicationController
```
Comment (aya-soft) [inline app/controllers/admin/set_confirm_email_controller.rb]: Delete in favour to Admin::UsersController#update

File: app/controllers/admin/set_admin_controller.rb [hunk 2]
```diff
@@ -0,0 +1,19 @@
+class Admin::SetAdminController < ApplicationController
```
Comment (aya-soft) [inline app/controllers/admin/set_admin_controller.rb]: Delete in favour to Admin::UsersController#update

File: app/controllers/admin/users_controller.rb [hunk 3]
```diff
@@ -1,20 +1,16 @@
...
   end
 
   def create
-    @user = User.new
-    set_date_for_confirmation!(params[:user])
-    if @user.update_attributes(user_params)
-      flash[:notice] = "User created"
-      redirect_to admin_users_path
+    @user = User.new(user_params)
+    if @user.save
```
Comment (aya-soft) [inline app/controllers/admin/users_controller.rb]: Add before save confirmation_at

File: app/controllers/admin/users_controller.rb [hunk 4]
```diff
@@ -26,10 +22,8 @@ def edit
 
   def update
     @user = User.find(params[:id])
-    set_date_for_confirmation!(params[:user])
     if @user.update_attributes(user_params)
-      flash[:notice] = "changes applied"
-      redirect_to admin_users_path
+      redirect_to admin_users_path, notice: "User updated"
     else
       render 'edit'
```
Comment (aya-soft) [inline app/controllers/admin/users_controller.rb]: :edit maybe better

File: config/routes.rb [hunk 5]
```diff
@@ -4,6 +4,8 @@
 
   namespace :admin do
     resources :users
+    resources :set_confirm_email, only: [:update, :destroy]
```
Comment (aya-soft) [inline config/routes.rb]: Please, remove

File: app/controllers/admin/users_controller.rb (hunk 3 above)
Comment (BigiDigi89) [reply app/controllers/admin/users_controller.rb]: resolved with merge

**Pull request merged**